includes/admin/upgrades/upgrade-functions.php

```diff
@@ -1353,19 +1352,9 @@ function edd_upgrade_render_v30_migration() {
 	$addresses = $wpdb->get_results( "SELECT umeta_id FROM {$wpdb->usermeta} WHERE meta_key = '_edd_user_address' LIMIT 1" );
```

Comment:
Since we no longer need to check if these are empty we can avoid a lot of extra queries by removing the initial value checks.

Reply:
@spencerfinnell I've merged in `issue/6810-2` as well as removed queries for data, have a look over that please.